pkg/storage/analytics.go

```diff
@@ -0,0 +1,81 @@
+package storage
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/dgraph-io/badger/v2"
+)
+
+type Analytics struct {
+	InstallID            string    `json:"install_id"`
+	RunID                string    `json:"run_id"`
+	Version              string    `json:"version"`
+	Timestamp            time.Time `json:"timestamp"`
+	UploadIndex          int       `json:"upload_index"`
+	GOOS                 string    `json:"goos"`
+	GOARCH               string    `json:"goarch"`
+	GoVersion            string    `json:"go_version"`
+	MemAlloc             int       `json:"mem_alloc"`
+	MemTotalAlloc        int       `json:"mem_total_alloc"`
+	MemSys               int       `json:"mem_sys"`
+	MemNumGC             int       `json:"mem_num_gc"`
+	BadgerMain           int       `json:"badger_main"`
+	BadgerTrees          int       `json:"badger_trees"`
+	BadgerDicts          int       `json:"badger_dicts"`
+	BadgerDimensions     int       `json:"badger_dimensions"`
+	BadgerSegments       int       `json:"badger_segments"`
+	ControllerIndex      int       `json:"controller_index" kind:"cumulative"`
```

Comment:
wasn't really sure what to name this key. I've struck with kind

Reply:
Yeah, this is good, and I like this approach of using struct tags.